src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,60 @@
-""" This is a dummy example """
+import requests
+import json
+
+COORDINATES = {
+        "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+        "London": {"latitude": 51.507351, "longitude": -0.127758},
+        "Rio": {"latitude": -22.906847, "longitude": -43.172896},
+    }
+VARIABLES = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
+API_URL = "https://api.open-meteo.com/v1/forecast"
+
+# Función auxiliar genérica para hacer llamadas a APIs
```

Comment:
If a comment do not add clarity then should be removed. Each function should be document with a combination of:
- Clear name
- Clear parameters names
- Parameters type hints.
- Docstring if needed

Reply:
Keep everything in english. You never know who will end up reading your code.